apps/nestjs-wallet/test/vc-api/exchanges/rebeam/rebeam-cpo-node.ts

```diff
@@ -0,0 +1,66 @@
+import { plainToClass } from 'class-transformer';
+import { ExchangeDefinitionDto } from '../../../../src/vc-api/exchanges/dtos/exchange-definition.dto';
+import { VpRequestInteractServiceType } from '../../../../src/vc-api/exchanges/types/vp-request-interact-service-type';
+import { VpRequestQueryType } from '../../../../src/vc-api/exchanges/types/vp-request-query-type';
+
+export class RebeamCpoNode {
+  #exchangeId = `b229a18f-db45-4b33-8d36-25d442467bab`;
```

Comment:
```suggestion
  private exchangeId = `b229a18f-db45-4b33-8d36-25d442467bab`;
```
It's not a JavaScript.

Reply:
It is JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields
So it's actually better to use than `private` because `private` is a TS feature whereas this as a JS feature

Reply:
It does not convince me, maybe it make more sens for ICL, but for Nest app?

Reply:
I was convinced by this article https://www.executeprogram.com/blog/typescript-features-to-avoid
See "Avoid the private keyword" section.

Reply:
Is this a good blog, I've never heard of them before. They are also saying not to use Enums and decorators :sweat_smile:

Reply:
Yes, I think their argument for both of those though is that enums and decorators require the typescript compiler to generate extract javascript code (so the JS doesn't "just work" if you remove the Typescript syntax). Personally though, I'm not as concerned by that as they are.

However, their guidance on `#` for private fields makes alot of sense to me but they are basically saying that the feature is in Js, so why not use it.

Do you see it differently?

Reply:
> I think we still should add a `holder` to the VP

Thanks for the reminder, I will add before merging